Remove warning about backups on multi-node (#786)
timescale/docs: timescaledb/overview/core-concepts/backup-restore.md

Physical backups may also be used with Write-Ahead Log (WAL) archiving to
achieve an ongoing backup.

> In multi-node deployments, care should be taken to ensure that one can
> restore to a point that ensures consistency across all nodes. Such a
> restore point can be created with the function
> [`create_distributed_restore_point`][create_distributed_restore_point],
> and later used when restoring from a physical backup.

Reviewer: ```suggestion
If you have a multi-node deployment, make sure you can restore to a 
point that ensures consistency across all nodes. You can create a 
restore point with the
[`create_distributed_restore_point`][create_distributed_restore_point] 
function, and use it later when you restore from a physical backup.
```
